google/cloud/bigtable/README.md

```diff
@@ -42,7 +42,40 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
 
 ## Release Notes
 
-### v0.9.x - TBD
+### v0.10.x - TBD
+
+### v0.9.x - 2019-05
+
+* **Breaking Changes**
+  * Return `google::cloud::future` from `InstanceAdmin` functions.
+  * Remove unused `bigtable::GrpcError`.
+  * Remove Snapshot-related functions, tests, examples, etc.
+* Continue to implement more async APIs (**Note: These are not yet stable**):
+  * `InstanceAdmin`: implement `AsyncSetIamPolicy`, `AsyncDeleteAppProfile`,
+    `AsyncGetAppProfile`, `AsyncCreateAppProfile`, `AsyncGetIamPolicy`,
+    `AsyncTestIamPermissions`, `AsyncUpdateAppProfile`, `AsyncUpdateInstance`,
+    `AsyncCreateCluster`, `AsyncUpdateCluster`, `AsyncCreateInstance`,
+    `AsyncListAppProfiles`.
+  * `TableAdmin`: Implement `AsyncGenerateConsistencyToken`,
+    `AsyncCheckConsistency`, `AsyncListTables`, `AsyncDropAllRows`,
+    `AsyncDropRowsByPrefix`.
+  * `Table`: Implement `AsyncReadModifyWriteRow`.
+* Bugfixes:
+  * Need `ignore_warnings` to actually delete an AppProfile.
+  * Fix portability/logical errors in shell scripts.
+  * Fix a race condition in `MutationBatcher`.
+* Implemented a number of missing code samples.
+* Cleanups:
+  * Remove `noex::Table*` dependencies in `Table::BulkApply()`,
+    `Table::*ReadModifyWriteRow()`.
+  * Remove `noex::Table::ReadModifyWriteRow()` and `SampleRows()`.
+  * Remove unused `AsyncFutureFromCallback`.
+* Test-only changes:
```

Comment:
In the past @coryan has [suggested](https://github.com/googleapis/google-cloud-cpp/pull/2142#pullrequestreview-210420336) removing any test-related changes, because customers probably don't care about them.